File: python-learning/while.py
ulangi =  input("Are you ready? ")
print("Start")
counter = 0

while ulangi:
    jawab = input("Apakah anda mau mengulangi? (yes/no) : ")  #input yes/no
    counter += 1
    if jawab =="no":
        ulangi = False
    print("Pengulangan ke-" + str(counter))

File: javascript-learning/part-2/while.js
const prompt = require('prompt-sync')({sigint : true});

var ulangi =  prompt("Are you ready?");
console.log("Start")
var counter = 0;

while(ulangi){
    var jawab = prompt("Apakah mau mengulangi? (yes/no) : ")
    counter++;
    if(jawab == false || jawab == "no"){
        ulangi = false;
    }
    console.log("Pengulangan ke-"+counter)

}